admin_site/bibos_admin/settings.py

```diff
@@ -63,11 +62,12 @@
 # http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
 # although not all choices may be available on all operating systems.
 # In a Windows environment this must be set to your system time zone.
-TIME_ZONE = env('TIME_ZONE')
+# Timezone/Language
+TIME_ZONE = 'CET'
```

Comment:
I understand why we might want to hard-code the language to the only one we actually support, but why should we do that for time zones?

Reply:
You're right. I'll look at a sane default.

Reply:
Addressed in commit 0c5a1da417af2234e8b037565f6f1b8a958b8463.